build.js

```diff
@@ -0,0 +1,7 @@
+let arg = process.argv.slice(2).join(' ');
+
+const execSync = require('child_process').execSync;
+
+execSync('npm run clean');
+execSync('npm run pre-build -- ' + arg, {stdio:[0, 1, 2]});
+execSync('node index.js ' + arg, {stdio:[0, 1, 2]});
```

Comment:
I think we could do this in Bash, rather than in javascript.

Reply:
This was done solely because with npm commands there was no apparent way to pass arguments to the pre-build script if it was done this way: `npm run clean && npm run pre-build && node index.js`

This exact case was discussed here in much detail and this was the solution recommended in the end: https://stackoverflow.com/a/51401577/8816118